test/cljc/quil/test_util.cljc

```diff
@@ -0,0 +1,12 @@
+(ns quil.test-util
+  #?(:clj (:import java.awt.GraphicsEnvironment)))
+
+(defn- display-density []
+  #?(:clj (.. GraphicsEnvironment (getLocalGraphicsEnvironment) (getDefaultScreenDevice) (getScaleFactor))
+     :cljs (.-devicePixelRatio js/window)))
```

Comment:
I don't think you actually need cljs version here as display-density used only by `path-to-snippet-snapshots` and that function is called only from clj test.

Reply:
Yes it's not used but it was bugging me to have a cljc with no cljs implementation, not really sure why :) In fact it's a smell because thinking about it what I wanted was just a way to use the logic from both `snippet.clj` and `test_server.clj`. This shouldn't require cljc at all. What do you think about this approach instead: https://github.com/quil/quil/pull/311/commits/d4f91e29b70e20b80cf4138cf7673a099e4ddc38

Reply:
Sounds good. I also prefer not having cljc unless it's really need to be shared between both languages.